app/grandchallenge/reader_studies/views.py

```diff
@@ -511,6 +512,28 @@ def get_success_url(self):
         return self.reader_study.get_absolute_url()
 
 
+class AddGroundTruthViaAnswersToReaderStudy(
+    BaseAddObjectToReaderStudyMixin, FormView
+):
+    form_class = GroundTruthCopyAnswersForm
+    template_name = "reader_studies/ground_truth_copy_answers_form.html"
+    type_to_add = "Ground Truth"
+
+    def get_form_kwargs(self):
+        kwargs = super().get_form_kwargs()
+        kwargs.update({"reader_study": self.reader_study})
+        return kwargs
+
+    def form_valid(self, form):
+        is_valid = super().form_valid(form)
+        if is_valid:
+            form.create_ground_truth()
+        return is_valid
```

Comment:
`form_valid` is only called when the form is valid. `super().form_valid(form)` does not tell you if the form is valid or not, it returns a redirect response. There is no need to check it as it is always a non-empty string, so this can be simplified:

```suggestion
        response = super().form_valid(form)
        
        form.create_ground_truth()
        
        return is_valid
```

Reply:
Ah, I had missinterpreted its relation to the `is_valid`. If the form_valid execution is already dependent on the form being valid this is fine.